styles/overrides/_styleguidist.scss

```diff
@@ -197,6 +197,13 @@ $rsgSearchHeaderWidth: 250px;
         margin-top: $xlarge-space;
     }
 
+    [class*="rsg--tab-"] {
+        // Fix WEB-1861: fix scroller issue if long text that doesn't break
+        .CodeMirror-line {
+            word-break: break-word;
```

Comment:
It would be nice if we could preserve side scrolling, and preventing lines from breaking to new lines.

Can you give the solution in [this Codepen](https://codepen.io/jeffkamo/pen/MQaxdz) a shot? Let me know if you want me to run you through the fix.

It may require setting `pre.CodeMirror-line` elements to `white-space: pre` (they're currently `pre-wrap`)

Reply:
Nice solution, that worked 👍